### ISSUE TITLE
No indication that uploading from K6 to Elastic is failed because of wrong credentials

### ISSUE THREAD

**Issue by MurzNN** [elastic/xk6-output-elasticsearch#13]
I'm trying to configure the correct access credentials for Elastic, but it's a bit problematic, because if I put obviously incorrect credentials like this:
```
export K6_ELASTICSEARCH_CLOUD_ID="XXXXXX3f4254280ba2b209985ad4bf7:dXMtY2VudHJhbDEuZ2NwLmNsb3VkLmVzLmlvOjQ0MyQ1MTQ5MjgwODJmYjA0ODYxOTFjYzhjNzIzOTU4NTU2ZCRiN2FlMmU0YzgxMmE0YWQ2YTAwZGEzZjBjMTMyXXXXXX=="
export K6_ELASTICSEARCH_USER=foo
export K6_ELASTICSEARCH_PASSWORD=bar
./k6 run script.js -o output-elasticsearch
```
I see no errors in the output, it just finishes correctly.

Could you please implement error reporting, when the upload is failed? Thanks!

**Comment by danielmitterdorfer** [comment]
Thanks for the report! I've fixed this in #14. If k6 can't connect to Elasticsearch for whatever reason you'll get the following now upon startup (in the example below credentials are wrong):

```
ERRO[0000] could not create the 'output-elasticsearch' output: cannot connect to Elasticsearch (status code 401) 
```

**Comment by MurzNN** [comment]
Thanks, tested it and it works well!
